### PR TITLE
Sec 3

### PR DESCRIPTION
**Pull request by Anbuganeshpdmr**

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -1,5 +1,10 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hello world!   22");
+
+        System.out.println("Hello world!   23");
+        System.out.println("Hello world!   24");
```
Comment (AnbuGanesh26) [inline src/Main.java:8]: Remove 24!!

**Review by AnbuGanesh26 — CHANGES_REQUESTED** [review]
Please see the reviews

**Review by Anbuganeshpdmr — COMMENTED** [review]
changed

**Review by AnbuGanesh26 — APPROVED** [review]
OK

**Pull request merged**